controllers/pattern_controller.go

```diff
@@ -511,3 +555,124 @@ func (r *PatternReconciler) actionPerformed(p *api.Pattern, reason string, err e
 	}
 	return r.onReconcileErrorWithRequeue(p, reason, err, nil)
 }
+
+func (r *PatternReconciler) createGitServer() (string, error) {
```

Comment:
@mbaldessari I feel like this function, `generateCustomChartValues()`, and many of the `const`s are an argument for an intermediate chart hosted by us that depends on the upstream one (although security folks may still not like that), has a bunch of these defaults, and creates the secrets, routes, etc.  Would be much less work for the operator. WDYT?